otel-integration/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## OpenTelemtry-Integration
 
+### v0.0.87 / 2024-07-18
+- [FEAT] Add process option to hostmetrics preset to scrape process metrics.
```

Comment:
Would be nice some documentation about this preset.
1. Metrics Exposed
2. Expected Cardinality
3. Use Case

I think it would be useful, don't need to be in this PR